include/z64cutscene.h

```diff
@@ -3,286 +3,478 @@
+    /* 0x3D */ CS_DEST_ZORAS_DOMAIN_CREDITS,
+    /* 0x3E */ CS_DEST_KOKIRI_FOREST_CREDITS_PART_1,
+    /* 0x3F */ CS_DEST_KOKIRI_FOREST_CREDITS_PART_2,
+    /* 0x40 */ CS_DEST_HYRULE_FIELD_CREDITS,
+    /* 0x41 */ CS_DEST_LON_LON_RANCH_CREDITS_ALT, // unused
+    /* 0x42 */ CS_DEST_KAKARIKO_VILLAGE_FROM_TRAIL_OWL,
+    /* 0x43 */ CS_DEST_HYRULE_FIELD_FROM_LAKE_HYLIA_OWL,
+    /* 0x44 */ CS_DEST_CUTSCENE_MAP_DEKU_SPROUT_PART_2,
+    /* 0x45 */ CS_DEST_KOKIRI_FOREST_DEKU_SPROUT_PART_3,
+    /* 0x46 */ CS_DEST_DEATH_MOUNTAIN_TRAIL_CREDITS_PART_2,
+    /* 0x47 */ CS_DEST_TEMPLE_OF_TIME_CREDITS,
+    /* 0x48 */ CS_DEST_ZELDAS_COURTYARD_CREDITS,
+    /* 0x49 */ CS_DEST_LON_LON_RANCH_CREDITS_PART_1,
+    /* 0x4A */ CS_DEST_LON_LON_RANCH_CREDITS_PART_2,
+    /* 0x4B */ CS_DEST_LON_LON_RANCH_CREDITS_PART_3,
+    /* 0x4C */ CS_DEST_LON_LON_RANCH_CREDITS_PART_4,
+    /* 0x4D */ CS_DEST_LON_LON_RANCH_CREDITS_PART_5,
+    /* 0x4E */ CS_DEST_LON_LON_RANCH_CREDITS_PART_6,
+    /* 0x4F */ CS_DEST_LON_LON_RANCH_1, // unused
+    /* 0x50 */ CS_DEST_LON_LON_RANCH_2, // unused
+    /* 0x51 */ CS_DEST_LON_LON_RANCH_3, // unused
+    /* 0x52 */ CS_DEST_LON_LON_RANCH_4, // unused
+    /* 0x53 */ CS_DEST_LON_LON_RANCH_5, // unused
+    /* 0x54 */ CS_DEST_LON_LON_RANCH_6, // unused
+    /* 0x55 */ CS_DEST_LON_LON_RANCH_7, // unused
+    /* 0x56 */ CS_DEST_LON_LON_RANCH_8, // unused
+    /* 0x57 */ CS_DEST_LON_LON_RANCH_9, // unused
+    /* 0x58 */ CS_DEST_LON_LON_RANCH_10, // unused
+    /* 0x59 */ CS_DEST_LON_LON_RANCH_11, // unused
+    /* 0x5A */ CS_DEST_LON_LON_RANCH_12, // unused
+    /* 0x5B */ CS_DEST_LON_LON_RANCH_13, // unused
+    /* 0x5C */ CS_DEST_LON_LON_RANCH_14, // unused
+    /* 0x5D */ CS_DEST_LON_LON_RANCH_15, // unused
+    /* 0x5E */ CS_DEST_LON_LON_RANCH_FROM_EPONAS_SONG,
+    /* 0x5F */ CS_DEST_STONES_RESTORED_CONDITIONAL, // unused
+    /* 0x60 */ CS_DEST_DESERT_COLOSSUS_FROM_BLUE_WARP,
+    /* 0x61 */ CS_DEST_GRAVEYARD_FROM_BLUE_WARP,
+    /* 0x62 */ CS_DEST_DEATH_MOUNTAIN_CRATER_FROM_BLUE_WARP,
+    /* 0x63 */ CS_DEST_SACRED_FOREST_MEADOW_FROM_BLUE_WARP,
+    /* 0x64 */ CS_DEST_KOKIRI_FOREST_FROM_BLUE_WARP,
+    /* 0x65 */ CS_DEST_DESERT_COLOSSUS_FROM_NABOORU_CAPTURE,
+    /* 0x66 */ CS_DEST_TEMPLE_OF_TIME_FRONT_OF_PEDESTAL, // unused
+    /* 0x67 */ CS_DEST_HYRULE_FIELD_TITLE_SCREEN,
+    /* 0x68 */ CS_DEST_TITLE_SCREEN_DEMO,
+    /* 0x69 */ CS_DEST_GRAVEYARD_SUNS_SONG_PART_2,
+    /* 0x6A */ CS_DEST_ROYAL_FAMILYS_TOMB_SUNS_SONG_PART_3,
+    /* 0x6B */ CS_DEST_GANONS_CASTLE_DISPEL_FOREST_BEAM,
+    /* 0x6C */ CS_DEST_GANONS_CASTLE_DISPEL_WATER_BEAM,
+    /* 0x6D */ CS_DEST_GANONS_CASTLE_DISPEL_SHADOW_BEAM,
+    /* 0x6E */ CS_DEST_GANONS_CASTLE_DISPEL_FIRE_BEAM,
+    /* 0x6F */ CS_DEST_GANONS_CASTLE_DISPEL_LIGHT_BEAM,
+    /* 0x70 */ CS_DEST_GANONS_CASTLE_DISPEL_SPIRIT_BEAM,
+    /* 0x71 */ CS_DEST_GANONS_CASTLE_DISPEL_BARRIER_CONDITONAL,
+    /* 0x72 */ CS_DEST_HYRULE_FIELD,
+    /* 0x73 */ CS_DEST_HYRULE_FIELD_FROM_IMPA_ESCORT,
```

Comment:
I guess
`CS_DEST_HYRULE_FIELD_IMPA_ESCORT` is "into the cutscene after impa escorts Link outside"
and `CS_DEST_HYRULE_FIELD_FROM_IMPA_ESCORT` is "after impa flash bangs hers way out" ?

could that be clarified somehow?